tidb-cloud/import-parquet-files.md

```diff
@@ -23,44 +23,61 @@ You can import both uncompressed and Snappy compressed [Apache Parquet](https://
 > - `ARRAY`
 > - `MAP`
 
-1. If a Parquet file is larger than 256 MB, consider splitting the file into smaller files, each with a size around 256 MB.
+1. If a Parquet file is larger than 256 MB, consider splitting it into smaller files, each with a size around 256 MB.
 
     TiDB Cloud supports importing very large Parquet files but performs best with multiple input files around 256 MB in size. This is because TiDB Cloud can process multiple files in parallel which can greatly improve the import speed.
```

Comment:
```suggestion
    TiDB Cloud supports importing very large Parquet files but performs best with multiple input files around 256 MB in size. This is because TiDB Cloud can process multiple files in parallel, which can greatly improve the import speed.
```